src/polymer/polymer-element.ts

```diff
@@ -184,8 +184,8 @@ export class ScannedPolymerElement extends ScannedElement implements
   }
 
   resolve(document: Document): PolymerElement {
-    this.applyJsdocDemoTags(document.url);
-    return resolveElement(this, document);
+    const element = new PolymerElement();
```

Comment:
Looks like resolveElement is gone now in master.  There was apparently a big refactoring since this PR opened.  Time for a re-do.